src/v2/Components/BankDebitForm/BankDebitProvider.tsx

```diff
@@ -5,31 +5,63 @@ import { getENV } from "v2/Utils/getENV"
 import { BankDebitForm } from "./BankDebitForm"
 import { CreateBankDebitSetupForOrder } from "./Mutations/CreateBankDebitSetupForOrder"
 import { BankAccountPicker_order } from "v2/__generated__/BankAccountPicker_order.graphql"
+import createLogger from "v2/Utils/logger"
+import { Message, Spacer, Text } from "@artsy/palette"
 
 interface Props {
   order: BankAccountPicker_order
 }
 
 const stripePromise = loadStripe(getENV("STRIPE_PUBLISHABLE_KEY"))
 
+const logger = createLogger("Order/Routes/Payment/index.tsx")
+
+const BankSetupErrorMessage = () => {
+  return (
+    <>
+      <Message
+        title="Bank transfer is not available at the moment"
+        variant="error"
+      >
+        <Text variant="sm">
+          Refresh the page or select another payment method.
+        </Text>
+      </Message>
+      <Spacer mt={2} />
```

Comment:
@rquartararo - A good habit for working with layouts is to always handle outer margin and padding outside of the component. Since `Message` has all of our common spacer values mixed in ([via `Flex`](https://github.com/artsy/palette/blob/main/packages/palette/src/elements/Message/Message.tsx#L42) you can use the `MessageProps` type: 

```tsx
const BankSetupErrorMessage: React.FC<MessageProps> = props => {
  return (
    <Message {...props}>...</Message>
  )
}
```
and then later
```tsx
<BankSetupErrorMessage mt={2} />
```

Reply:
But an even simpler way to do this (while preserving the literate coding style) would be to assign `Message` it to a variable:

```tsx
const BankSetupErrorMessage = Message
```
or skip the intermediate variable altogether and just use `<Message>` in the main body of the code.